packages/basic/tests/init/utils/date-time/old-basic/README.md

```
@@ -0,0 +1,569 @@
+    INIT --> INIT_UTILS
+    
+    UTILS --> COOKIE
+    UTILS --> STORAGE
+    UTILS --> REQUEST
+    UTILS --> ROUTE
+    UTILS --> ENCODE
+```
+
+### 核心目录结构
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)